include/maliput/routing/route.h

```diff
@@ -233,6 +232,25 @@ class Route final {
   /// @throws common::assertion_error When @p start_position is not valid.
   api::LaneSRoute ComputeLaneSRoute(const api::RoadPosition& start_position) const;
 
+  /// Evaluates whether this Route is connected end to end.
+  ///
+  /// End to end connectivity for a Route implies the following conditions:
+  /// - The first Phase contains only one start position.
+  /// - The last Phase contains only one end position.
+  /// - The number of end positions in a given Phase and start positions in the following Phase are the same.
+  /// - For each position in the end positions in a given Phase there is a one-to-one relation in the start
```

Comment:
Can:

> position in the end positions in a given Phase

be simplified to be:

> end position in a given Phase

?

Reply:
[5f99c1c](https://github.com/maliput/maliput/pull/628/commits/5f99c1c613ff4ae3206bf0796b0287698f0d08ae)